kg_microbe/transform_utils/prefixmap.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
will anything else need to be added here? I think any used ontology?
if so, that would include:
CHEBI, NCBITaxonomy, ENVO, GO, UBERON, MONDO, HPO